docs/community/developer-guide.mdx

```diff
@@ -277,3 +277,55 @@ with a relational database, that will enable us to
 never read out credentials, when sending data to the client.
 
 :::
+
+## InfoCard
+When making something in homarr, if you wish to add some information for the user, you may do so using the InfoCard. An element preprogrammed and easy to implement.
```

Comment:
Change something to a more specific term. 
Maybe this should be in a "helper components" section or similar.